packages/docker-node/src/interface.ts

```diff
@@ -0,0 +1,15 @@
+import { config } from '@ckb-lumos/lumos'
+
+export interface CKBNode {
+  url: string
+  port: string
+  host: string
+  lumosConfig: config.Config
```

Comment:
Why does ckb node's interface depend on `lumos`

Reply:
These property is not the dependencies for the Node, they provider the Lumos config which is from the Node.